examples/with-cypress-typescript/README.md

```diff
@@ -0,0 +1,21 @@
+# Cypress + TypeScript Next.js example
+
+This is a really simple project that shows the usage of Next.js with [Cypress](https://cypress.io) and TypeScript.
```

Comment:
Let's make sure to add a more detailed description here. Like what kind of tests should I be expecting by running this example, why is cypress useful here, e.t.c